currency-rates/arch-go.yaml

```diff
@@ -0,0 +1,53 @@
+version: 1
+threshold:
+  compliance: 100
+  coverage: 75
+dependenciesRules:
+  - package: "**.services.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.repo.**"
+        - "**.tools.**"
+  - package: "**.clients.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.tools.**"
+  - package: "**.handlers.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.tools.**"
+  - package: "**.repo.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.tools.**"
+  - package: "**.routers.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.handlers.**"
+        - "**.tools.**"
+  - package: "**.scheduler.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.tools.**"
+  - package: "**.services.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.repo.**"
+        - "**.tools.**"
+  - package: "**.tools.**"
+    shouldOnlyDependsOn:
+      internal:    
+  - package: "**.tests.**"
+    shouldOnlyDependsOn:
+      internal:    
+  - package: "**.cmd.**"
```

Comment:
You package cmd can depend on anything, because it's entrypoint of your application, and it can get any deps it needed to run your app.

Reply:
Got your point, done